### PR TITLE
[ADD] l10n_in: add information about PAN field in india

### PR DESCRIPTION
**Pull request by vivp-odoo**
This PR adds the information about PAN field and GSTIN in company form view.

task- 3930028

### CODE REVIEW

File: content/applications/finance/fiscal_localizations/india.rst [hunk 1]
```diff
@@ -46,6 +46,14 @@ localization:
 
 .. _india/e-invoicing:
 
+Indian Configuration
+====================
+
+In :menuselection:`Settings --> Users & Companies --> Companies`, add your PAN and GST number.
```
Comment (dade-odoo) [inline content/applications/finance/fiscal_localizations/india.rst:52]:
```suggestion
In :menuselection:`Settings --> Users & Companies --> Companies`, add your :guilabel:`PAN` and
:guilabel:`GSTIN` to ensure the correct chart of accounts and taxes are installed.
```

File: content/applications/finance/fiscal_localizations/india.rst [hunk 2]
```diff
@@ -46,6 +46,14 @@ localization:
...
+
+.. image:: india/company-form.png
+   :alt: Company form
```
Comment (dade-odoo) [inline content/applications/finance/fiscal_localizations/india.rst:55]:
```suggestion
```

File: content/applications/finance/fiscal_localizations/india.rst (hunk 1 above)
Comment (dade-odoo) [reply content/applications/finance/fiscal_localizations/india.rst:52]: We use guilabels when talking about an interactive element such as a field to complete, and I suggested a little more information to give the reader more context as to why it's important.

File: content/applications/finance/fiscal_localizations/india.rst (hunk 2 above)
Comment (dade-odoo) [reply content/applications/finance/fiscal_localizations/india.rst:55]: We try to limit the number of screenshots and only include them when they're really necessary, so I suggest removing this one (don't forget to delete the actual file as well).

**Review by dade-odoo — CHANGES_REQUESTED** [review]
Hi @vivp-odoo , thanks for this addition to the Indian localization. I made a few quick suggestions and then we can get it merged.

File: content/applications/finance/fiscal_localizations/india.rst (hunk 1 above)
Comment (vivp-odoo) [reply content/applications/finance/fiscal_localizations/india.rst]: > We use guilabels when talking about an interactive element such as a field to complete, and I suggested a little more information to give the reader more context as to why it's important.

Hello @dade-odoo , I have updated the information with correct reason for readers understanding.
thanks!!

**Review by dade-odoo — APPROVED** [review]
@vivp-odoo great, thanks! Passing you on the final review and r+ :)

File: content/applications/finance/fiscal_localizations/india.rst [hunk 3]
```diff
@@ -46,6 +46,13 @@ localization:
 
 .. _india/e-invoicing:
 
+Indian Configuration
+====================
+
+In :menuselection:`Settings --> Users & Companies --> Companies`, add your :guilabel:`PAN` and
+:guilabel:`GSTIN`. The :guilabel:`PAN` is essential for determining the type of taxpayer,
+while :guilabel:`GSTIN`is required for generating e-Invoices and E-waybills.
```
Comment (xpl-odoo) [inline content/applications/finance/fiscal_localizations/india.rst:54]:
```suggestion
:guilabel:`GSTIN`. The PAN is essential for determining the type of taxpayer,
while GSTIN is required for generating e-Invoices and E-waybills.
```
Guilabels aren't necessary in this case, and the second guilabel caused a _ci/documentation_ error due to a missing space after the second backtick :)

**Review by xpl-odoo — CHANGES_REQUESTED** [review]

File: content/applications/finance/fiscal_localizations/india.rst (hunk 3 above)
Comment (vivp-odoo) [reply content/applications/finance/fiscal_localizations/india.rst]: Hello @xpl-odoo 
made the suggested changes.

**Review by xpl-odoo — APPROVED** [review]
You can r+ whenever ready ;)
@robodoo delegate+

**Comment by vivp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**